### ISSUE TITLE
send data to the server faster

### ISSUE THREAD

**Issue by GreenPedal** [schollz/find3#196]
is there  a way to make the  Passive tracking to sniff more often?? when i go into a room it takes a couple for sec to notice im there.. i would love to know when im in the room

**Comment by nash-stokes** [comment]
Are you running from the command line? If so, you can modify the scan time.

**Comment by GreenPedal** [comment]
yes i am.. i put it in a service... so what kind of line to i need to put in it to make the scan time faster

**Comment by nash-stokes** [comment]
So when you execute the initial command, you pass it several flags. The number after the scantime flag dictates how many seconds between scans.

**Comment by GreenPedal** [comment]
so when i do this sudo ./find3-cli-scanner -i wlan0 -device DEVICE -family FAMILY \
    -server https://cloud.internalpositioning.com \
    -scantime 40 -forever -passive -no-modify 

your saying th scantime 40 if i change that to say 1 it will scan every one sec??

**Comment by nash-stokes** [comment]
From my experience, yes.

**Comment by GreenPedal** [comment]
do you know if i can do less then 1 sec... or is that the lowest

**Comment by nash-stokes** [comment]
Not sure. I'd suggest trial and error. I never went smaller than 5 seconds. It may cause issues on his server since the creator is the one hosting that server.

**Comment by GreenPedal** [comment]
oh im host the sever..

**Comment by nash-stokes** [comment]
Sorry, I was looking at the server in the command you posted. I'd try it out. What devices are you using for your scanning network?

**Comment by GreenPedal** [comment]
raspberry pi zore w with PAU06 i have like 7 of them all over the house..

**Comment by jaburges** [comment]
came here to ask this - is there a way to track a phone passively with around 1 second updates? 

Can't figure out from the doc how quickly you can scan - and what the results are like with 3 or 4 RPis around the house

**Comment by IoTThinks** [comment]
> came here to ask this - is there a way to track a phone passively with around 1 second updates?
> 
> Can't figure out from the doc how quickly you can scan - and what the results are like with 3 or 4 RPis around the house

I think no point to track every 1 second.
How fast a human can move in 1 second? 2m?
The accuracy may not be up to 2m also.
